config/urls.py

```diff
@@ -67,6 +69,9 @@ class BulkNestedDefaultRouter(BulkRouterMixin, NestedDefaultRouter):
 documents_router.register(
     "modifications", ModificationViewSet, basename="modifications"
 )
+router.register(
```

Comment:
That sounds good. I thought it might confuse existing users of the API because having the same entity either under `documents` or at the root level is a subtle distinction, but you know the users way better than I do.